metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Analysis.java

```diff
@@ -118,32 +118,37 @@
     @Column(nullable = false, columnDefinition = "TEXT")
     private String description;
 
-    @ApiModelProperty(position = 5, dataType = "java.lang.String", notes = "Url to a Study")
+    @ApiModelProperty(position = 5, dataType = "java.lang.String", notes = "Url to a Project")
+    @JsonProperty
+    @ManyToOne(optional = true) // TODO jmmut: I'm putting this as optional until we complete the project import from DB
```

Comment:
That's fine, but we must ensure there is a separate ticket for that in the backlog

Reply:
EVA-2012